src/main/java/chess/domain/board/Board.java

```diff
@@ -0,0 +1,86 @@
+package chess.domain.board;
+
+import chess.domain.piece.Empty;
+import chess.domain.piece.Piece;
+import chess.domain.piece.PieceColor;
+import chess.domain.position.Position;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class Board {
+
+    private static final Piece EMPTY_PIECE = new Empty();
+    private final Map<Piece, Position> coordinates;
```

Comment:
```suggestion
    private static final Piece EMPTY_PIECE = new Empty();
    
    private final Map<Piece, Position> coordinates;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가한다면 가독성이 상승합니다